### PR TITLE
Harmonize yaml formatting

### LINKED ISSUE #19: Get rid of "incomplete final line found" warning
Introduced in [this](https://github.com/krlmlr/actions-sync/commit/174f4830802e679eff0d3c18d10242ab2f341fb1) commit in the file `R-CMD-check-dev.yaml`.

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #19

### CODE REVIEW

File: R-CMD-check-dev.yaml [hunk 1]
```diff
@@ -148,4 +148,4 @@ jobs:
 
       - uses: ./.github/workflows/check
         with:
-          results: ${{ matrix.package }}
```
Comment (IndrajeetPatil) [inline R-CMD-check-dev.yaml]: This will fix #19

**Comment by krlmlr** [comment]
Thanks. This needs to be applied to all branches simultaneously. What did you use for normalizing the files?

**Comment by IndrajeetPatil** [comment]
> What did you use for normalizing the files?

VS Code's JSON formatter.

> This needs to be applied to all branches simultaneously. 

Oh? I thought this'd will be merged in the `base` first, and then be cherry-picked and applied to all branches?

**Comment by krlmlr** [comment]
The problem here is that cherry picking is unlikely to work. I suspect there will be too many conflicts, but you can try locally.

**Comment by IndrajeetPatil** [comment]
Okay, in that case, I will mark this as a draft for now.

But can I just directly push to `base` the only important change from this PR, which is to get rid of the hanging newline in `R-CMD-check-dev.yaml`, which is causing failures (#19)?

**Comment by krlmlr** [comment]
I'm afraid pushing to `base` alone won't help, you need to change `base` and then cherry-pick into all branches.

**Comment by IndrajeetPatil** [comment]
> I'm afraid pushing to `base` alone won't help, you need to change `base` and then cherry-pick into all branches.

Sorry for the confusing phrasing. That's what I meant to do!

**Comment by krlmlr** [comment]
Thinking about it, it *might* work if you revert the changes to the "matrix" section in R-cmd-check.yaml . Is there a way to teach the VS Code formatter to ignore a code block?

**Comment by IndrajeetPatil** [comment]
> it might work if you revert the changes to the "matrix" section in R-cmd-check.yaml 

I have reverted these changes.

**Pull request merged**